libmscore/dynamic.cpp

```diff
@@ -144,6 +172,10 @@ void Dynamic::read(XmlReader& e)
                   _velocity = e.readInt();
             else if (tag == "dynType")
                   _dynRange = Range(e.readInt());
+            else if (tag == "veloChange")
+                  _changeInVelocity = e.readInt();
+            else if (tag == "veloChangeSpeed")
+                  _velChangeSpeed = Speed(e.readInt());
```

Comment:
Problem is, this is written using `ScoreElement::writeProperty`, which means that it can only be written as an int, due to the way the framework is set up. So, as with any property that takes an enum value, it is stored as an int.

Reply:
After some further investigation, that turns out to not be true. Some enums are stored with custom `P_TYPE` that's specifically for them. `P_TYPE::HEAD_GROUP` is one example, it turns out that the way that is written is defined in xmlwriter.cpp, in the function at line 177. So, I'll try and implement writing the speed as a string.

Reply:
Fixed in latest commit (rebased and force-pushed).